cms/templatetags/cms_wizard_tags.py

```diff
@@ -11,6 +12,7 @@
 register = template.Library()
 
 
+@register.tag(name='cms_wizard')
```

Comment:
was this template tag documented? If not then no need for deprecation cycle and can be removed.